WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -333,12 +333,12 @@ public function process_matched_token( $stackPtr, $group_name, $matched_content
 	 * @param string $function Function name.
```

Comment:
```suggestion
	 * @param string $function_name Function name.
```

Reply:
Done. Thanks.

(I only realized after I pulled this PR we hadn't fixed the names in this codebase before 🙄)